crates/bevy_ecs/src/schedule/condition.rs

```diff
@@ -13,6 +13,22 @@ pub type BoxedCondition<In = ()> = Box<dyn ReadOnlySystem<In = In, Out = bool>>;
 /// Implemented for functions and closures that convert into [`System<Out=bool>`](crate::system::System)
 /// with [read-only](crate::system::ReadOnlySystemParam) parameters.
 ///
+/// # Marker type parameter
+///
+/// `Condition` trait has `Marker` type parameter, which has no special meaning,
+/// but exists to work around the limitation of Rust's trait system.
+///
+/// Type parameter in return type can be omitted by calling [`IntoSystem::into_system`],
+/// but usually have to be specified when passing a condition to a function.
```

Comment:
I don't see a button to suggest code edits.

Better replace "can be omitted" with something like "can be set to `<()>`".